src/main/flight/mixer.c

```diff
@@ -424,44 +425,79 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
 #endif
 
 static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnabled) {
-    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
-    const float motorMixDelta = 0.5f * motorMixRange;
+    float motorMixNormalizationFactor;
+    float airmodeTransitionPercent;
+    float motorDeltaScale;
+
+    if (airmodeEnabled || throttle > 0.5f) {
+        airmodeTransitionPercent = 1.0;
+        motorDeltaScale = 0.5f;
+    } else {
+        // this scales the motor mix authority to be 0.5 at 0 throttle, and 1.0 at 0.5 throttle as airmode off intended for things to work.
+        // also lays the groundwork for how an airmode percent would work.
+        airmodeTransitionPercent = scaleRangef(throttle, 0.0f, 0.5f, 0.5f, 1.0f); // 0.5 throttle is full transition, and 0.0 throttle is 50% airmodeTransitionPercent
+        motorDeltaScale = 0.5f * airmodeTransitionPercent; // this should be half of the motor authority allowed
+    }
+
+    motorMixNormalizationFactor = motorMixRange > 1.0 ? airmodeTransitionPercent / motorMixRange : airmodeTransitionPercent;
```

Comment:
maybe  define `motorMixNormalizationFactor ` this as `const float` here, not as `float` up above?